src/controller/data_model/gen/CHIPClientCallbacks.h

```diff
@@ -158,6 +156,3 @@ typedef void (*TvChannelTvChannelListListAttributeCallback)(void * context, uint
 typedef void (*TargetNavigatorTargetNavigatorListListAttributeCallback)(void * context, uint16_t count,
                                                                         _NavigateTargetTargetInfo * entries);
 typedef void (*TestClusterListInt8uListAttributeCallback)(void * context, uint16_t count, uint8_t * entries);
-typedef void (*TestClusterListOctetStringListAttributeCallback)(void * context, uint16_t count, chip::ByteSpan * entries);
-typedef void (*TestClusterListStructOctetStringListAttributeCallback)(void * context, uint16_t count,
-                                                                      _TestListStructOctet * entries);
```

Comment:
Seems like unrelated changes.

Reply:
I don't know why this happened. I can only tell that a more up to date ZAP Tool seems to remove this lines.

Reply:
It happened because of the pieces removed from  src/controller/data_model/controller-clusters.zap